components/api/src/main/java/com/hotels/styx/api/ClientCookieDecoder.java

```diff
@@ -0,0 +1,350 @@
+/*
+  Copyright (C) 2013-2020 Expedia Inc.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+  http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+ */
```

Comment:
Do we need to maintain the original copyright header here?

Reply:
Changed to the original header. I actually thought that was the header included as I just copied the file.